docs/mddocs/Quickstart/ollama_quickstart.md

```diff
@@ -227,3 +228,9 @@ If you meet this error, please check your Linux kernel version first. You may en
 
 #### 8. Save GPU memory by specify `OLLAMA_NUM_PARALLEL=1`
 If you have a limited GPU memory, use `set OLLAMA_NUM_PARALLEL=1` on Windows or `export OLLAMA_NUM_PARALLEL=1` on Linux before `ollama serve` to reduce GPU usage. The default `OLLAMA_NUM_PARALLEL` in ollama upstream is set to 4.
+
+#### 9. `cannot open shared object file` error when executing  `ollama serve`

```

Comment:
I feel it's a same issue as 3 ?

Reply:
Can you just reuse 3 instead of creating a new one ? or maybe we just modify 3 ?

Reply:
I think they are different errors (although with the same reason), keeping section 3 could be helpful for users that are still running earlier versions.

Reply:
Please also update related Chinese version.